### PR TITLE
NTRN-303 add basic slinky bindings docs

### PR DESCRIPTION
**Pull request by quasisamurai**
[TASK](https://hadronlabs.atlassian.net/browse/NTRN-303)

This PR implements some basic docs for bindings, related to `Oracle` & `MarketMap` modules

### CODE REVIEW

File: docs/neutron/modules/3rdparty/skip/slinky/overview.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+# Overview
+We have integrated the [Slinky](https://skip-protocol-docs.netlify.app/slinky/overview) modules (Oracle & MarketMap) into the Neutron and added WASM bindings to facilitate their usage. This integration enhances our blockchain with reliable, real-time price feeds and market data.
+Using `neutron-sdk` you can query these modules via contracts
+
+### Oracle
+For example usage of these queries in contract, check our related [dev-contract](https://github.com/neutron-org/neutron-dev-contracts/blob/727aa94dff53373e34226d31d5bdcfc66e3a1aaf/contracts/oracle/src/contract.rs#L53)
```
Comment (pr0n00gler) [inline docs/neutron/modules/3rdparty/skip/slinky/overview.md:6]: Let's do not refer to dev-contract in the docs.

We usually refer to simple and easily readable contracts in neutron-sdk repo

File: docs/neutron/modules/3rdparty/skip/slinky/overview.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
+# Overview
+We have integrated the [Slinky](https://skip-protocol-docs.netlify.app/slinky/overview) modules (Oracle & MarketMap) into the Neutron and added WASM bindings to facilitate their usage. This integration enhances our blockchain with reliable, real-time price feeds and market data.
```
Comment (pr0n00gler) [inline docs/neutron/modules/3rdparty/skip/slinky/overview.md:2]: Weird link, why not `https://docs.skip.money/slinky/overview`?

File: docs/neutron/modules/3rdparty/skip/slinky/overview.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+pub struct GetPricesResponse {
+    /// A list of price responses for the requested currency pairs.
+    pub prices: Vec<GetPriceResponse>,
+}
+``` 
+
+### Market Map
+For more detailed descriptions, refer to the [MarketMap README](https://github.com/skip-mev/slinky/blob/main/x/marketmap/README.md).
+
+For example usage of these queries in contract, check our related [dev-contract](https://github.com/neutron-org/neutron-dev-contracts/blob/feat/sdk-50/contracts/marketmap/src/contract.rs#L52)
```
Comment (pr0n00gler) [inline docs/neutron/modules/3rdparty/skip/slinky/overview.md:78]: The same about dev-contracts

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

**Review by sotnikov-s — CHANGES_REQUESTED** [review]
please attach a link to the task and add some description to the PR

**Review by sotnikov-s — CHANGES_REQUESTED** [review]

File: sidebars.js [hunk 4]
```diff
@@ -284,10 +284,17 @@ const sidebars = {
...
+                                            label: 'Slinky',
+                                            type: 'category',
+                                            items: [
+                                                'neutron/modules/3rdparty/skip/slinky/overview',
+                                            ]
+                                        }
+                                    ],
+                                },
+                                {
+                                    label: "Cosmos Hub",
```
[the remark this answers is not in the document]
Comment (sotnikov-s) [reply sidebars.js:297]: and the bindings pages are not included in the sidebars.js

File: docs/neutron/modules/3rdparty/skip/slinky/bindings/messages.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+## Oracle
+For example usage of these queries in contract, check our related [example contract](https://github.com/neutron-org/neutron-sdk/tree/d9abe67f0f62d4ea42d1943af53189ec6674d29e/contracts/marketmap)
+```rust
+   pub enum OracleQuery {
+   GetAllCurrencyPairs {},
+   GetPrice { currency_pair: CurrencyPair },
+   GetPrices { currency_pair_ids: Vec<String> },
+   }
+   ```
```
Comment (sotnikov-s) [inline docs/neutron/modules/3rdparty/skip/slinky/bindings/messages.md:11]: there is a number of unformatted snippets. please apply formatting:

image

File: docs/neutron/modules/3rdparty/skip/slinky/bindings/messages.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+```
+
+### GetPrice
+Get the price of a specific pair
+
+Request:
+```rust
+GetPrice { currency_pair: CurrencyPair }
+```
+resp:
```
Comment (sotnikov-s) [inline docs/neutron/modules/3rdparty/skip/slinky/bindings/messages.md:41]:
```suggestion
Response:
```

**Pull request closed without merging**